src/components/SignUp/Privacy.tsx

```diff
@@ -1,48 +1,119 @@
 import React from 'react';
+import styled from 'styled-components';
 import Container from 'react-bootstrap/Container';
 import Row from 'react-bootstrap/Row';
 import Col from 'react-bootstrap/Col';
 import { Link } from 'react-router-dom';
-import yellow_blob from '../../images/yellow_blob_1.svg';
+import yellow_blob from '../../images/yellow_blob_2.svg';
 import Image from 'react-bootstrap/Image';
+import { fonts } from '../../theme/styleVars';
 
 const Privacy: React.FC<{
   setForm: any;
   formData: any;
 }> = props => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  const { setForm, formData } = props;
+  const { formData } = props;
 
   return (
     <Container>
       <Row>
         <Col lg="8">
-          <h3>Hi, {formData.basicsFirstName}!</h3>
-          <h1>LET'S TALK PRIVACY</h1>
-          <h2>Your privacy is our top concern. Always.</h2>
-          <p>
+          <TitleThree>Hi, {formData.basicsFirstName}!</TitleThree>
+          <Title>LET'S TALK PRIVACY</Title>
+          <Tagline>Your privacy is our top concern. Always.</Tagline>
+          <PrivacyPar>
             We take privacy seriously (seriously). We will never share, sell, or
             otherwise distribute your information to a third party. We also know
             that some of the information we’re asking for is personal, and for
             some could be sensitive in nature. The reason we’re asking is to
             create the most effective, inclusive, and equitable casting & hiring
-            platform possible. If you see “Private” next to a field, that’s
-            because even though we’re collecting the information to inform our
-            search algorithm, we won’t display this information on your profile,
-            or to producers and casting directors. If you’d like to learn more
-            about the measures we take to secure your data,{' '}
+            platform possible. If you see <span>“Private”</span> next to a
+            field, that’s because even though we’re collecting the information
+            to inform our search algorithm, we won’t display this information on
+            your profile, or to producers and casting directors. If you’d like
+            to learn more about the measures we take to secure your data,{' '}
             <Link to="/faq">click here</Link> [link to FAQs].
-          </p>
+          </PrivacyPar>
           <Link to="/terms-of-service">
```

Comment:
Instead of having a separate component called `Caption`, I would simply make an extension of Link that looks like this:
```
const TermsPrivacyLink = styled(Link)`
  color: ${colors.veryDarkGrayBlue};
  font-family: ${fonts.lora};
  font-size: 14px;
  margin-bottom: 18px;
  margin-top: 26px;
`;
```

Note: you'll need to add the color to our color variables

Then, you can just do:

```
<TermsPrivacyLink to="/terms-of-service">View our full terms and privacy policy</TermsPrivacyLink>
```

Reply:
Definitely overlooked just creating a styled link